packages/front-end/components/SavedGroupForm.tsx

```diff
@@ -88,7 +88,10 @@ const SavedGroupForm: FC<{
           value={rawText}
           onChange={(e) => {
             setRawText(e.target.value);
-            form.setValue("groupList", e.target.value.trim().split(","));
```

Comment:
I was trimming the beginning and end of the entire CSV string instead of the individual values